libs/wingcompiler/src/compile.ts

```diff
@@ -213,7 +213,7 @@ export async function compile(entrypoint: string, options: CompileOptions): Prom
   });
 
   try {
-    vm.runInContext(artifact, context);
+    await vm.runInContext(artifact, context);
```

Comment:
Is this to change handle async code from somewhere?

Reply:
isn't needed at the end- removed, thanks for noticing!